CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.2)
+cmake_minimum_required(VERSION 3.1)
```

Comment:
Since we don't use any CMake 3.2 feature, we can relax the required version to 3.1. 3.1. is the version where `CMAKE_CXX_STANDARD` variable was introduced.